src/ext/java/org/opentripplanner/ext/gtfsgraphqlapi/datafetchers/TripImpl.java

```diff
@@ -372,6 +374,17 @@ public DataFetcher<GraphQLWheelchairBoarding> wheelchairAccessible() {
     return environment -> GraphQLUtils.toGraphQL(getSource(environment).getWheelchairBoarding());
   }
 
+  @Override
+  public DataFetcher<TripOccupancy> occupancy() {
+    return environment -> {
+      Trip trip = getSource(environment);
+      TripPattern pattern = getTransitService(environment).getPatternForTrip(trip);
+      return new TripOccupancy(
+        getRealtimeVehiclesService(environment).getVehicleOccupancyStatus(pattern, trip.getId())
+      );
```

Comment:
I wonder if we should move this logic into the RealtimeVehicleService, but I'm not sure. Seems like a good discussion for the dev meeting.

Reply:
Another thing we could discuss that I've thought about for years already is that do we really need the fuzzyTripMatching parameter for all these updaters or could we just use it when required. I think there is only a minimal extra cost from creating the matcher once.